pkg/manifests/config.go

```diff
@@ -173,17 +182,18 @@ type RemoteWriteSpec struct {
 }
 
 type PrometheusK8sConfig struct {
-	LogLevel            string                               `json:"logLevel"`
-	Retention           string                               `json:"retention"`
-	NodeSelector        map[string]string                    `json:"nodeSelector"`
-	Tolerations         []v1.Toleration                      `json:"tolerations"`
-	Resources           *v1.ResourceRequirements             `json:"resources"`
-	ExternalLabels      map[string]string                    `json:"externalLabels"`
-	VolumeClaimTemplate *monv1.EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate"`
-	RemoteWrite         []RemoteWriteSpec                    `json:"remoteWrite"`
-	TelemetryMatches    []string                             `json:"-"`
-	AlertmanagerConfigs []AdditionalAlertmanagerConfig       `json:"additionalAlertmanagerConfigs"`
-	QueryLogFile        string                               `json:"queryLogFile"`
+	LogLevel              string                               `json:"logLevel"`
+	Retention             string                               `json:"retention"`
+	NodeSelector          map[string]string                    `json:"nodeSelector"`
+	Tolerations           []v1.Toleration                      `json:"tolerations"`
+	Resources             *v1.ResourceRequirements             `json:"resources"`
+	ExternalLabels        map[string]string                    `json:"externalLabels"`
+	VolumeClaimTemplate   *monv1.EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate"`
+	RemoteWrite           []RemoteWriteSpec                    `json:"remoteWrite"`
+	TelemetryMatches      []string                             `json:"-"`
+	AlertmanagerConfigs   []AdditionalAlertmanagerConfig       `json:"additionalAlertmanagerConfigs"`
+	QueryLogFile          string                               `json:"queryLogFile"`
+	EnforcedBodySizeLimit string                               `json:"enforcedBodySizeLimit,omitempty"`
```

Comment:
Can you add a comment about how using the parameter?
* an empty value means no enforcement
* "automatic" means that CMO picks up a value based on the cluster capacity
* A fixed size can be defined too.